data/customers-dynamodb.json

```diff
@@ -3,10 +3,10 @@
         "S": "AnyCompany Water"
     },
     "Valuation": {
-        "N": 10
+        "N": "10"
```

Comment:
I noticed this as well. When copy/pasting the content of this file into DynamoDB itself to add new items, the syntax validation was complaining about the quotes. Thanks for fixing this.